lib/services/namespace.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package services
+
+import (
+	"fmt"
+
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/trace"
+)
+
+// NamespaceSpecSchema is JSON schema for NameSpaceSpec
+const NamespaceSpecSchema = `{
+	"type": "object",
+	"additionalProperties": false,
+	"default": {}
+  }`
+
+// NamespaceSchemaTemplate is JSON schema for the Namespace resource
+const NamespaceSchemaTemplate = `{
+	"type": "object",
+	"additionalProperties": false,
+	"default": {},
+	"required": ["kind", "spec", "metadata"],
+	"properties": {
+	  "kind": {"type": "string"},
+	  "version": {"type": "string", "default": "v1"},
+	  "metadata": %v,
+	  "spec": %v
+	}
+  }`
+
+// GetNamespaceSchema returns namespace schema
```

Comment:
```suggestion
// GetNamespaceSchema returns Namespace schema
```
to make it consistent across other go-docs.